packages/core/src/lib/select/searchable-select.svelte

```diff
@@ -282,40 +344,53 @@ const handleKeydown = createHandleKey({
         aria-selected={isSelected}
         aria-checked={isChecked}
         aria-label={isOther
-          ? [otherOptionPrefix, option].filter(Boolean).join(' ')
-          : option}
+          ? [otherOptionPrefix, optionDisplayValue(option)]
+              .filter(Boolean)
+              .join(' ')
+          : optionDisplayValue(option)}
         class={cx(
-          'flex h-7.5 w-full cursor-pointer items-center justify-start text-xs',
```

Comment:
Why did we lose the explicit height? Figma spec's a fixed height of `30px`. Trying to rely on padding to get the proper height will result in pain, in my experience

Reply:
We lost the explicit height because we want wrap instead of horizontal scroll on the items

(talked offline  -- there was some confusion about what element this was on)